stat.go

```diff
@@ -85,9 +85,5 @@ const (
 	s_ISVTX = uint32(sshfx.ModeSticky)
 )
 
-// Legacy export:
-//
-// Go defines S_IFMT on windows, plan9 and js/wasm as 0x1f000 instead of
-// 0xf000. None of the the other S_IFxyz values include the "1" (in 0x1f000)
-// which prevents them from matching the bitmask.
+// S_IFMT is a legacy export. `sshfx.ModeType` should be used instead
```

Comment:
The problem is that the `sshfx` package is `internal` and so no one else can import it. So, the given alternative isn’t useful.

I would say that the local OS might use a different value from what we need to use internally, and there is no reason anyone should need this specific internal value.

We can also merge, and then I can adjust it after.

Reply:
Ah I see. That makes more sense. Better to have you leave the comment anyways. Thanks again